hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/cmd/FeedCommProcessor.java

```diff
@@ -243,13 +246,13 @@ public void onClose(int reasonCode, String reason) {
         // We also don't try to get another one if we were explicitly told to disconnect.
         if (!(disconnectCode == reasonCode && disconnectReason.equals(reason))) {
             switch (reasonCode) {
-                case 1008: { // VIOLATED POLICY - don't try again since it probably will fail again (bad credentials?)
-                    break;
-                }
-                default: {
-                    startReconnectJobThread();
-                    break;
-                }
+            case 1008: { // VIOLATED POLICY - don't try again since it probably will fail again (bad credentials?)
```

Comment:
hawkular coding standards indent case stmts inside switch - I just double checked and alerts and metric code as well a bus all indent case stmts inside switch (i.e. case and switch are not on the same indent level).